wgpu-core/src/device/global.rs

```diff
@@ -260,7 +260,15 @@ impl Global {
             };
 
             let (id, resource) = fid.assign(Arc::new(buffer));
-            api_log!("Device::create_buffer({desc:?}) -> {id:?}");
+            api_log!(
+                "Device::create_buffer({:?}{}) -> {id:?}",
+                desc.label.as_deref().unwrap_or(""),
+                if desc.mapped_at_creation {
+                    ", m-a-c"
```

Comment:
I'd never be able to figure out that `m-a-c` means "mapped at creation", why not spell it out

Reply:
That's more than fair.